microsoft-edge/webview2/release-notes.md

```diff
@@ -143,7 +288,7 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
 
 ---
 
-#### Bug fixes
+#### <a id="bug-fixes-1466-prerelease"/>Bug fixes
 
 *   Fixed a bug in which the custom header title in print settings could be wrong. ([Issue #2093](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2093))
 *   Display `AllowedCertificateAuthorities` in `add_ClientCertificateRequested` event as a `Base64` string. (Runtime) ([Issue #2346](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2346))

```

Comment:
near line 449, change from
```
```
to:
```
#### <a id="bug-fixes-1414-prerelease"/>Bug fixes
```
Make sure you're on the correct "Bug fixes" heading, below/inside of the h2:
```
## 1.0.1414-prerelease
```
which is line ~349.